app/main.ts

```diff
@@ -472,3 +497,188 @@ function calculateLocation(address: any) {
+    //click handler for addresses 
+    const tableClickHandler = function(e: Event) {
+        e.preventDefault;
+        e.stopPropagation;
+        (e.currentTarget as Element ).id = "esri-a11y-last-focus";
+        let id = this.getAttribute("data-id");
+        displayFeatureInfo(id, featureResults);
+    };
+    const tableFocusHandler = function(e: Event) {
+        const target = e.currentTarget as Element;
+        let featureName = target.firstElementChild.innerHTML;
+        // detail read of table items
+        liveDetailsNode.innerHTML = featureName;
+    }
+
+    //click handler for table toggler
+    const toggleBtn = document.getElementById("esri-a11y-table-toggle");
+    const toggleClickHandler = function(e: Event) {
+        e.preventDefault;
+        e.stopPropagation;
+        const tableContainer = document.getElementById("esri-a11y-table-component");
+        let containerClasses = tableContainer.classList;
+        let tableRows = tableContainer.getElementsByClassName("esri-a11y-map-table-row");
+        if(!containerClasses.contains("open")) {
+            tableContainer.classList.add("open");
+            for (var i = 0; i < tableRows.length; i++){
+                tableRows[i].setAttribute("tabIndex", "0");
+                tableRows[i].addEventListener('click', tableClickHandler, false);
+                tableRows[i].addEventListener('focus', tableFocusHandler, false);
+            }
+            //add keyboard handlers
+            tableContainer.addEventListener("keyup", tableKeyHandlers);
+
+            
+            //move focus to table
+            (tableRows[0] as HTMLElement).focus({preventScroll:true});
+            
+            // directions of how to use table and show live region
+            liveNode.classList.remove("hidden");
+            document.getElementById("intro").innerHTML = `Use the up and down arrow keys to navigate the table and find features. Use the enter key to toggle more information on the feature. 
+            To return to the map, press escape.`;
+            
+        } else {
+            tableContainer.classList.remove("open");
+            for (var i = 0; i < tableRows.length; i++){
+                tableRows[i].setAttribute("tabIndex", "-1");
+                tableRows[i].removeEventListener('click', tableClickHandler);
+                tableRows[i].removeEventListener('focus', tableFocusHandler);
+            }
+            //remove key handlers
+            tableContainer.removeEventListener("keyup", tableKeyHandlers);
+
+             // remove directions of how to use table and hide live region
+             document.getElementById("intro").innerHTML = "";
+             liveDetailsNode.innerHTML = "";
+             liveNode.classList.add("hidden");
+
+            //move focus back to toggle button
+            toggleBtn.focus();
+        }
+    };
+    //Hitting enter triggers click of button
+    var toggleEnterHandler = function(e: KeyboardEvent) : void {
```

Comment:
There are a few instances of var in app - replace with let or const as appropriate.

Reply:
Handled in 11314bd